dev-libs/miniaudio/miniaudio-0.11.21.ebuild

```
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Single file library for audio playback and capture"
+HOMEPAGE="https://miniaud.io"
+SRC_URI="https://github.com/mackron/miniaudio/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( public-domain MIT-0 )"
```

```suggestion
LICENSE="|| ( Unlicense MIT-0 )"
```
This also affects the other versions in ::gentoo. Maybe you could fix that as well :+1: